avogadro/core/gaussianset.h

```diff
@@ -66,6 +66,11 @@ class AVOGADROCORE_EXPORT GaussianSet : public BasisSet
   ~GaussianSet() AVO_OVERRIDE;
 
   /**
+   * Virtual copy constructor.
+   */
+  GaussianSet* clone() const AVO_OVERRIDE {return new GaussianSet(*this);};
```

Comment:
Don't need the final ;, please add a little white space such that

{ return new GaussianSet(*this); }